### PR TITLE
PSFR-771: Added aria-label and visually hidden text to Friday warning to provide context for non-visual users

### PR DESCRIPTION
**Pull request by ashliecodes**

### CODE REVIEW

File: server/views/macros/prisoners-table.njk [hunk 1]
```diff
@@ -32,19 +32,28 @@
     </td>
     <td class="govuk-table__cell" id="releaseDate">
       <span class="no-wrap">{{ releaseDate | formatDate }}</span> <br/>
-      {% if releaseDate | isFriday %}<span class="govuk-body-s" id="fridayRelease">⚠️ Friday</span> <br/>{% endif %}
+            {% if releaseEligibilityDate | isFriday %}
```
Comment (JamesBoobier) [inline server/views/macros/prisoners-table.njk:35]:
```suggestion
      {% if releaseEligibilityDate | isFriday %}
        <span class="govuk-body-s" id="fridayRelease" aria-hidden="true">⚠️ Friday <br/></span>
        <span class="govuk-visually-hidden"> Warning: release date is a Friday </span>
      {% endif %}
```

File: server/views/macros/prisoners-table.njk [hunk 2]
```diff
@@ -32,19 +32,28 @@
     </td>
     <td class="govuk-table__cell" id="releaseDate">
       <span class="no-wrap">{{ releaseDate | formatDate }}</span> <br/>
-      {% if releaseDate | isFriday %}<span class="govuk-body-s" id="fridayRelease">⚠️ Friday</span> <br/>{% endif %}
```
Comment (JamesBoobier) [inline server/views/macros/prisoners-table.njk:35]: Could you look at the formatting in this file?

File: server/views/partials/profileHeader.njk [hunk 3]
```diff
@@ -44,7 +44,8 @@
                 <br />
                 {% set isDayFriday = prisonerData.personalDetails.releaseDate | isFriday %}
                 {% if isDayFriday %}
-                  <span class="govuk-body-s">⚠️ (Friday)</span>
+                <span class="govuk-body-s" id="fridayRelease" aria-hidden="true">⚠️ Friday <br/></span>
```
Comment (JamesBoobier) [inline server/views/partials/profileHeader.njk:47]: Indent 2 spaces

File: server/views/macros/prisoners-table.njk [hunk 4]
```diff
@@ -32,19 +32,19 @@
...
+      {% if releaseEligibilityDate | isFriday %}<span class="govuk-body-s" id="fridayRelease" aria-hidden="true">⚠️ Friday <br/></span><span class="govuk-visually-hidden"> Warning: release date is a Friday </span>{% endif %}
```
Comment (JamesBoobier) [inline server/views/macros/prisoners-table.njk:35]: Should this be `releaseDate`?

**Review by mkerry-moj — APPROVED** [review]

**Pull request merged**